index.bs

```diff
@@ -5822,6 +5822,12 @@ Since certificates are sizable relative to the storage capabilities of typical a
 
 Note: In order to interoperate, user agents storing large blobs on authenticators using [[!FIDO-CTAP]] are expected to use the provisions detailed in that specification for storing [=large, per-credential blobs=].
 
+Note: [[!FIDO-CTAP]] authenticators only support this extension for [=discoverable credentials=],
+and might return an incompatibility error unless
```

Comment:
At the WebAuthn level, the error is probably very generic. Maybe drop "incompatibility" in case readers expect to be able to distinguish such an error?

Reply:
Yep, good point.